forest/src/cli/mod.rs

```diff
@@ -175,6 +176,11 @@ pub struct CliOpts {
         possible_values = &["mainnet", "calibnet"],
     )]
     pub chain: String,
+    // having such an option isn't possible since target for env_logger can be either stderr or std out, for now. See https://github.com/seanmonstar/pretty-env-logger/issues/52
+    // #[structopt(
+    //     help = "target file for logs"
```

Comment:
You can use the help text from the original issue: "Specify a filename into which logging should be appended."

Reply:
Fixed